datanode/sqlstore/migrations/0004_oracle_specs.sql

```diff
@@ -1,4 +1,9 @@
 -- +goose Up
 ALTER TABLE oracle_specs DROP COLUMN IF EXISTS signers;
 ALTER TABLE oracle_specs DROP COLUMN IF EXISTS filters;
-ALTER TABLE oracle_specs ADD COLUMN data JSONB NOT NULL;
\ No newline at end of file
+ALTER TABLE oracle_specs ADD COLUMN data JSONB NOT NULL default '{}';
```

Comment:
Any chance this could break queries to the database if the entities have non nullable fields?

Reply:
I think if the database migration script wanted the column to be not null, then the Go could would expect the value to be not null and therefore handle it correctly. All we're doing is assigning an empty JSON object to the field if we are given a null value. The problem would be elsewhere, i.e. why are we being given a null value for this data